packages/common/src/client/appeals-api-client.js

```diff
@@ -116,6 +117,44 @@ class AppealsApiClient {
 		return response.json();
 	}
 
+	/**
+	 * @param {string} lpaCode
+	 * @returns {Promise<AppealCaseWithAppellant[]>}
+	 */
+	async getAppealsCaseDataV2(lpaCode) {
+		const urlParams = new URLSearchParams();
+		urlParams.append('lpa-code', lpaCode);
+		const endpoint = `${v2}/appeal-cases?${urlParams.toString()}`;
+		const response = await this.#makeGetRequest(endpoint);
+		return response.json();
+	}
```

Comment:
This and `getDecidedAppealsCaseDataV2` could have been merged and add a `decidedOnly` parameter - but also fine to have them separate with a name - it does make the intention clear.